web/static/js/components/stage_progression_button.jsx

```diff
@@ -111,6 +121,7 @@ StageProgressionButton.defaultProps = {
 const mapStateToProps = reduxState => ({
   reduxState,
   retroUpdateRequested: reduxState.retro.updateRequested,
+  votes: reduxState.votes,
```

Comment:
Followed the prior art in the line above, but we could also pull this out into a selector if preferable?

Reply:
we can leave these as as, as there's not much computation/logic to these lines.